store/v2/commitment/store.go

```diff
@@ -563,6 +563,9 @@ loop:
 					node.Value = []byte{}
 				}
 			}
+			if node.Version == 0 {
```

Comment:
Not sure why an empty version is possible in the first place. Maybe there is a better fix